db/migrate/20171009043433_change_min_rank_type.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeMinRankType < ActiveRecord::Migration
+  def change
+    change_column :events, :min_title, :string, :default => Person::TITLES[Person::TITLES.length - 1]
```

Comment:
I would rather not have this default here. This would be set when the migration runs. If the list of Titles changes later, you could have a default that wasn't in the list. This is more likely in the future, as we move the titles into the department model.

Reply:
Understood! I'll fix it as soon as I've rebased! It shouldn't be long, but I'm a little new to this, so I'm moving tentatively...

Reply:
OK, I've removed the default!